Hyokune/bonfire
# Feature/#1 initialization

Closes: #1

How to test this PR:

1. Checkout branch locally
2. Check that application can be started locally
3. Profit
